### DISCUSSION TITLE
[2기-B] 이수진

### DISCUSSION THREAD

**Discussion by dltnwls9623** [super-corini/programmers_kdt_II#352] [2기 Monthly Project I]
## Monthly Project I  

World Happiness Report 2015 데이터에 대해 EDA를 진행하였습니다

- 주소 : http://sujinlee.pythonanywhere.com/
- 코드(배포용) : https://github.com/dltnwls9623/EDA-project
- 새로 공부한 것 : Plotly
- 아쉬운 점
  - 개인적인 시간이 많이 부족해서 더 많은 분석을 진행하지 못했습니다(다른 연도 데이터 등)
  - 그래프와 데이터에 대한 설명이나 background를 추가하고 싶었지만 이것 역시 시간 때문에..😥

**Comment by super-corini** [comment]
깔끔합니당 👍 한정된 시간에 최대의 효율을 내주신것이 보입니당
